docs/build.md

```diff
@@ -1,42 +1,43 @@
 # Build from source
 
-This document describes how to build an epoch node from source on current Ubuntu 16.04.4 LTS.
+This document describes how to build an epoch node from source on current Ubuntu 16.04.4 LTS or Ubuntu 18.04 LTS.
 The commands below assume you are logged in with `sudo` user.
 
 ## Dependencies installation
 
-### Tools and libraries
+### Common tools and libraries
 
-Make sure your Ubuntu version and it's packages are up to date, then install required tools and libs:
+Make sure your Ubuntu version and it's packages are up to date, then install required tools and libraries:
 ```bash
 sudo apt-get -qq update \
 && sudo apt-get -y upgrade \
-&& sudo apt-get -qq -y install \
-git curl autoconf build-essential ncurses-dev libssl-dev
+&& sudo apt-get -qq -y install git curl autoconf build-essential ncurses-dev libssl-dev
 ```
 
 ### OTP install
 
-Required Erlang OTP version is `20.2.3` it could be installed from source running below commands:
+Required Erlang OTP version is `20.1`.
```

Comment:
Didn't follow the discussion, but 20.1 is a step backwards?

Reply:
@hanssv It's already what we require in [rebar.config](https://github.com/aeternity/epoch/blob/master/rebar.config#L19)